spec/dummy/application.rb

```diff
@@ -7,6 +7,8 @@
 
 module Dummy
   class Application < Rails::Application
+    CACHE_CONTROL_FIVE_MINUTES = "public, max-age=300".freeze
```

Comment:
Unfortunately, this workaround was required, as I couldn't get `ClimateControl` to override an `ENV` variable before this class was loaded.